### PR TITLE
Provide ESM and CJS versions

### PR DESCRIPTION
**Pull request by arlac77**
AS node now supports ESM and conditional exports.
We can provide CJS and ESM in one package

### CODE REVIEW

**Comment by manast** [comment]
thanks for the patch. Seems like the option you passed to tsc is not correct though, check the build status.

**Comment by arlac77** [comment]
Sorry I am not used to ts but the 4 errors where the beforehand too (at least on my local machine)

**Comment by manast** [comment]
In the latest test run all the tests related to child processors are failing because they do not find the processor files.

File: package.json [hunk 1]
```diff
@@ -2,14 +2,20 @@
...
+    "require": "./dist/cjs/index.js",
+    "./": "./"
+  },
   "source": "src/index.ts",
   "author": "Taskforce.sh Inc.",
   "license": "MIT",
   "scripts": {
     "docs": "typedoc --out docs src",
-    "build": "tsc && yarn copylua",
+    "build": "tsc && tsc --build tsconfig.esm.json && yarn copylua",
```
Comment (manast) [inline package.json:18]: do we need to build 2 times?

**Comment by manast** [comment]
closing since no answer to the feedback.

**Pull request closed without merging**